src/shared/components/studyTagsTooltip/StudyTagsTooltip.tsx

```diff
@@ -92,19 +97,54 @@ class StudyInfoOverlay extends React.Component<
                         )}
                     />
                 );
-                overlay =
-                    resultKeyLength > 0
-                        ? [
-                              description,
-                              <br />,
-                              <div className="studyTagsTooltip">
-                                  {' '}
-                                  <JsonToTable
-                                      json={this.studyMetadata.result}
-                                  />
-                              </div>,
-                          ]
-                        : description;
+                if (this.props.infoIcon === true) {
+                    overlay =
+                        resultKeyLength > 0
+                            ? [
+                                  description,
+                                  <br />,
+                                  <div className="studyTagsTooltip">
+                                      {' '}
+                                      <JsonToTable
+                                          json={this.studyMetadata.result}
+                                      />
+                                  </div>,
+                              ]
+                            : description;
+                } else {
+                    let message = getServerConfig()
+                        .skin_home_page_unauthorized_studies_global_message;
+                    let placeholders = message.match(/[{][$][^}]*[}]/g);
+                    let placeholdersReplaced: boolean = true;
+                    if (placeholders !== null) {
+                        let jp = require('jsonpath');
+                        placeholders.forEach(placeholder => {
+                            let placeholderReplaceValue = jp.query(
+                                this.studyMetadata.result,
+                                placeholder.replace(/["'{}]/g, '')
+                            );
+                            if (placeholderReplaceValue.length === 0)
+                                placeholdersReplaced = false;
+                            else
+                                message = message.replace(
+                                    placeholder,
+                                    placeholderReplaceValue
+                                );
+                        });
+                    }
```

Comment:
Move to utility function with name similar to replacePlacerholders(globalmessage)

Reply:
Fixed